### PR TITLE
Feature/remove res admin

### PR DESCRIPTION
**Pull request by Nicolas-Rodriguez-Ch**
## Summary

This pull request adds functionality to delete a restaurant admin and a second language to the remove admin feature. It also includes the following changes:

- Merged the `main` branch into `TableTop-FE`.
- Pulled changes from the `main` branch.
- Fixed indentation issues in the edit modal.

## Details

This commit implements the ability to delete a restaurant admin, which is a useful feature for maintaining the TableTop app. In addition, it adds support for a second language to the remove admin feature, which makes it more accessible to users who speak languages other than English.

The commit also includes several other changes, including merging the `main` branch into `TableTop-FE` and pulling the latest changes from the `main` branch. Additionally, the commit fixes an issue with the indentation in the edit modal.

### CODE REVIEW

File: src/components/EditModal/EditModal.jsx [hunk 1]
```diff
@@ -1,86 +1,96 @@
...
     e.preventDefault();
     const validationErrors = {};
     const form = e.target;
     const title = form.title.value;
     const price = form.price.value;
     const description = form.description.value;
 
-    if (title.length <= 1 || typeof title !== 'string') {
-      validationErrors.title = 'title must be at least 2 characters long';
+    if (title.length <= 1 || typeof title !== "string") {
```
Comment (ht1204) [inline src/components/EditModal/EditModal.jsx]: `if (!title.length || typeof title !== "string") {`

File: src/components/EditModal/EditModal.jsx [hunk 2]
```diff
@@ -1,86 +1,96 @@
...
+      validationErrors.title = "title must be at least 2 characters long";
     }
     if (price.length === 0) {
```
Comment (ht1204) [inline src/components/EditModal/EditModal.jsx]: `if (!price.length) {`

File: src/components/EditModal/EditModal.jsx (hunk 1 above)
Comment (Nicolas-Rodriguez-Ch) [reply src/components/EditModal/EditModal.jsx]: Resolved

File: src/components/EditModal/EditModal.jsx (hunk 2 above)
Comment (Nicolas-Rodriguez-Ch) [reply src/components/EditModal/EditModal.jsx]: Resolved

**Comment by Nicolas-Rodriguez-Ch** [comment]
Changes according to feedback

File: src/components/EditModal/EditModal.jsx (hunk 1 above)
Comment (ht1204) [reply src/components/EditModal/EditModal.jsx]: No, this is not done as expected, check the falsy value for `title.length`, check that the conditional should be:
`if (!title.length || typeof title !== "string") {`  => `!title.length`

File: src/components/EditModal/EditModal.jsx (hunk 2 above)
Comment (ht1204) [reply src/components/EditModal/EditModal.jsx]: Check that this should be: `if (!price.length) {`

**Comment by Nicolas-Rodriguez-Ch** [comment]
changes according to feedback

File: src/components/EditModal/EditModal.jsx (hunk 1 above)
Comment (Nicolas-Rodriguez-Ch) [reply src/components/EditModal/EditModal.jsx]: now it is resolved

File: src/components/EditModal/EditModal.jsx (hunk 2 above)
Comment (Nicolas-Rodriguez-Ch) [reply src/components/EditModal/EditModal.jsx]: now it is resolved

**Review by ht1204 — APPROVED** [review]

**Pull request merged**